kalite/distributed/templates/distributed/base.html

```diff
@@ -13,7 +13,7 @@
         <meta name="wot-verification" content="075abe861486e6bb3ab9"/>
         <meta name="viewport" content="width=device-width, initial-scale=1">
         {% block titletag %}
-        <title>{% block title %}{% trans title %}{% endblock title %} | KA Lite</title>
+        <title>{% block title %}{% trans "Welcome" %}{% trans title %}{% endblock title %} | KA Lite</title>
```

Comment:
It's a default so that pages without a title have something rather than " | KA Lite"